beluga_amcl/cmake/ament.cmake

```diff
@@ -21,13 +21,34 @@ find_package(rclcpp_components REQUIRED)
 find_package(rclcpp_lifecycle REQUIRED)
 find_package(std_srvs REQUIRED)
 
+add_library(ros2_common SHARED)
```

Comment:
@serraramiro1 nit: consider a slightly more descriptive name (one less likely to name clash with others). Perhaps `beluga_amcl_ros2_common`? I'm not great with names TBH.

Reply:
Also, I see it's a shared library but it's not installed.